pkg/log/zap/zap.go

```diff
@@ -142,8 +145,8 @@ type Options struct {
 	// Encoder configures how Zap will encode the output.  Defaults to
 	// console when Development is true and JSON otherwise
 	Encoder zapcore.Encoder
-	// TimeEncoder configures the zap time format. Defaults to epoch.
-	TimeEncoder zapcore.TimeEncoder
+	// EncoderConfigOptions configures options for Encoder.
```

Comment:
We should probably add a little more detail in the godocs for this option. 
And point out the caveat that @joelanford mentioned that this does not get applied when `Encoder != nil`.
```suggestion
	// EncoderConfigOptions can modify the EncoderConfig needed to initialize an Encoder.
	// See https://godoc.org/go.uber.org/zap/zapcore#EncoderConfig for the list of options 
	// that can be configured.
	// Note that the EncoderConfigOptions are not applied when the Encoder option is already set.
```